github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1015,7 +1015,7 @@ presubmits:
           automation/test.sh
         env:
         - name: TARGET
-          value: kind-1.28-wg-arm64
+          value: kind-1.31-wg-arm64
```

Comment:
env KUBEVIRT_PROVIDER is missing here.

Reply:
Is handled by automation/test.sh: https://github.com/kubevirt/kubevirt/blob/main/automation/test.sh#L104

Reply:
Yes, it handled by https://github.com/kubevirt/kubevirt/blob/main/automation/test.sh, and `KUBEVIRT_PROVIDER` will set automatically base on TARGET